kebechet/managers/update/update.py

```diff
@@ -798,6 +798,13 @@ def run(self, labels: list) -> typing.Optional[dict]:
                 comment=ISSUE_CLOSE_COMMENT.format(sha=self.sha),
             )
 
+            update_issue = self.sm.get_issue(_UPDATE_MERGE_REQUEST_TITLE)
```

Comment:
Not sure if this will work for GitHub and GitLab. IIRC GitLab treats merge requests and issues as separated entities (in opposite to GitHub).